cli/common/utils.go

```diff
@@ -489,6 +497,55 @@ func wrap(i, w int, s string) string {
 	return r
 }
 
+func ParseDependenciesFile(filepath string) ([]PackDependency, error) {
+	content, err := GetFileContent(filepath)
+	if err != nil {
+		return nil, err
+	}
+
+	lexer := stateful.MustSimple([]stateful.Rule{
+		{"Comment", `(?:#|//)[^\n]*\n?`, nil},
+		{"Ident", `[a-zA-Z]\w*`, nil},
+		{"Number", `(?:\d*\.)?\d+`, nil},
+		{"Punct", `[-[!@#$%^&*()+_={}\|:;"'<,>.?/]|]`, nil},
+		{"Whitespace", `[ \t\n\r]+`, nil},
+	})
+
+	parser := participle.MustBuild(
+		&PackDependency{},
+		participle.Lexer(lexer),
+		participle.Elide("Comment", "Whitespace"),
+	)
+
+	dependencies := []PackDependency{}
+	for _, line := range strings.Split(content, "\n") {
+		if line == "" {
+			continue
+		}
+
+		// We do this to process lines like: dependency>=2.4,<5
```

Comment:
Why don't you describe it in the grammar?
`PackDependency` structure can be a little more difficult: 

```
type DepRelation struct {
    Version string
    Relation string
}

type PackDependency struct {
	Name     string `@Ident`
	Relations []DepRelation
}
```